### PR TITLE
Feature/fix 131 truncate

### PR DESCRIPTION
**Pull request by davhin**
Fixes https://github.com/rocs-org/data-pipelines/issues/131

### CODE REVIEW

**Comment by davhin** [comment]
I moved the truncate query upstream so that is only executed once, independent of the insert. I then also remove the truncate_insert query builder since it serves no more use, and use the regular insert query instead

File: services/airflow/src/lib/dag_helpers/write_dataframe_to_postgres.py [hunk 1]
```diff
@@ -30,8 +31,15 @@ def connect_to_db_and_insert_pandas_dataframe(
 def connect_to_db_and_truncate_insert_pandas_dataframe(
     schema: str, table: str, data: pd.DataFrame
 ):
+    db_context = create_db_context()
+    sql_query = sql.SQL("TRUNCATE TABLE {schema}.{table};").format(
+        schema=sql.Identifier(schema),
+        table=sql.Identifier(table),
+    )
+    execute_sql(db_context, sql_query)
+    teardown_db_context(db_context)
```
Comment (jakobkolb) [inline services/airflow/src/lib/dag_helpers/write_dataframe_to_postgres.py:40]:
```suggestion
    _connect_to_db_and_execute(
        R.always(sql.SQL("TRUNCATE TABLE {schema}.{table};").format(
            schema=sql.Identifier(schema),
            table=sql.Identifier(table),
            )
        ), 
        R.T, 
        ""
    )
```

**Review by jakobkolb — CHANGES_REQUESTED** [review]
Also, add a test that writes more than 100 entries into the DB and makes sure that they actually end up there.

**Review by jakobkolb — APPROVED** [review]
lgtm!

**Comment by davhin** [comment]
> Also, add a test that writes more than 100 entries into the DB and makes sure that they actually end up there.

Done, First write adds 150 rows of data, second does 200

**Pull request merged**